languages/common-lisp/exercises/concept/sameness/.docs/hints.md

```diff
@@ -0,0 +1,19 @@
+## What is one supposed to do again?
```

Comment:
The formatting of this file is slightly different from the [formatting we expect tracks to use](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docshintsmd). The contents of this file are directly tied to my reply to https://github.com/exercism/v3/pull/1823/files#r453240873, regarding the defining of tasks. 

The [basics exercise](https://github.com/exercism/v3/blob/master/languages/common-lisp/exercises/concept/basics/.docs/hints.md) is a nice example of what the `hints.md` file should look like.

Regarding the actual hints, they look very nice! As far as I can tell you also don't give away the actual solution, which is great!

Reply:
Perhaps I need to think harder about how to define the 'tasks' for this exercise. Because currently I only see it having a single task. The student needs to choose the correct equality predicate and add a clause for it into a conditional expression. So a single task repeated multiple times.

This is part of the reason i wanted your review - I worry that the structure of this exercise may not be sufficiently "exercism-ish".

Reply:
I think the structure is totally fine! And in fact, I think creating several tasks might be easier than you think :) Consider your test suite:

```lisp
(test the-maze-of-numbers
  (is (eq 'victory (open-room :room-numbers #'robot)))
  (is (eq 'victory (open-room :room-number-of-different-types #'robot))))

(test the-maze-of-characters
  (is (eq 'victory (open-room :room-characters #'robot)))
  (is (eq 'victory (open-room :room-case-insensitive-chars #'robot))))

(test the-maze-of-strings
  (is (eq 'victory (open-room :room-strings #'robot)))
  (is (eq 'victory (open-room :room-case-insensitive-strings #'robot))))

(test the-maze-of-conses
  (is (eq 'victory (open-room :room-cons-of-symbols #'robot)))
  (is (eq 'victory (open-room :room-cons-of-chars #'robot)))
  (is (eq 'victory (open-room :room-cons-of-numbers #'robot)))
  (is (eq 'victory (open-room :room-cons-case-insensitive-chars #'robot)))
  (is (eq 'victory (open-room :room-cons-number-of-different-types #'robot))))

(test the-maze-of-arrays
  (is (eq 'victory (open-room :room-arrays #'robot)))
  (is (eq 'victory (open-room :room-arrays-looser-equal #'robot))))
```

To me, this looks like five different tasks :)

Reply:
I'm thinking of ways to perhaps _storify_ this a bit more based on the above subdivision (numbers/characters/strings/conses/arrays). Maybe @TheLostLambda has ideas?